### PR TITLE
Completed feature/new-sections

### PR DESCRIPTION
**Pull request by SiddheshShinde-tech**

### CODE REVIEW

File: css/sections.css [hunk 1]
```diff
@@ -21,3 +21,72 @@
   padding-top: 1rem;
   padding-bottom: 1rem;
 }
+
```
Comment (workeazee) [inline css/sections.css:24]: no shift+alt+f done

File: index.html [hunk 2]
```diff
@@ -209,6 +204,45 @@ <h1 class="main-heading">Pricing</h1>
...
+                    <h4 class="any-device-text">Tab / iPad</h4>
+                </div>
+              </div>
+          </div>
+          <div class="text-center">
+            <a href="https://play.google.com/store/apps/details?id=com.aummetals.workeazee">
+              <img class="playstore-logo" src="images/any-device-image/play.png" alt="Play Store Logo">
+            </a>
+          </div>
+          <div class="free-demo-class text-center">
```
Comment (workeazee) [inline index.html]: make this as a different container section

File: css/sections.css (hunk 1 above)
Comment (SiddheshShinde-tech) [reply css/sections.css:24]: Yeah ! Doing it

File: index.html (hunk 2 above)
Comment (SiddheshShinde-tech) [reply index.html]: Done. Please confirm with the new changes.

File: index.html [hunk 3]
```diff
@@ -209,6 +208,87 @@ <h1 class="main-heading">Pricing</h1>
...
+            >
+              <img
+                class="playstore-logo"
+                src="images/any-device-image/play.png"
+                alt="Play Store Logo"
+              />
+            </a>
+          </div>
+
+          <!-- FREE DEMO STARTS -->
```
Comment (workeazee) [inline index.html:262]: Remove the commented part from next PR.
There should be no code comments except from important msgs in code base

**Pull request merged**